web/src/components/MedicaidBusinessAreas.js

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import TextArea from './TextArea';
+import { MEDICAID_BUSINESS_AREAS_DISPLAY_LABEL_MAPPING } from '@cms-eapd/common';
+import { ChoiceList } from '@cmsgov/design-system';
+import { Controller, useFormContext } from 'react-hook-form';
+
+const MedicaidBusinessAreas = ({
```

Comment:
I'm sure you looked at this already, but it would be really nice to have just one component for this content since there's such a huge amount of code overlap with `ApdOverviewMMISFields.js`

Reply:
Going to incorporate in ApdOverviewMMISFields in [4574](https://app.zenhub.com/workspaces/enterprise-cmcseapd-6390a3073de2e96379767873/issues/gh/enterprise-cmcs/eapd/4574)